salt/_modules/metalk8s_service_configurator.py

```diff
@@ -0,0 +1,80 @@
+'''Metalk8s cluster and service configuration utility.'''
+
+import logging
+import re
+
+from salt.exceptions import CommandExecutionError
```

Comment:
Please respect pylint import suggestion
```
C:  8, 0: third party import "import yaml" should be placed before "from salt.exceptions import CommandExecutionError" (wrong-import-order)
```

Reply:
Maybe we should setup pylint for Salt tree (or `_modules` at least)?
That way we can enforce it in CI and focus our brain's bandwidth on functional issues.

Reply:
Agree we should but we need some extra stuff about "salt" stuffs like all the `__opts__` `__salt__` `__pillar__` ... but anyway agree we should have salt module checking in the CI so linting + unit test

Reply:
If this is something we really want to enforce, it would make sense to add it in CI. Especially since not everyone here uses `pylint` + the special Salt plugin to lint their Salt modules (I don't).

Reply:
I thin there is a pylint plugin for Salt, otherwise we can simply ignore Salt stuff by tweaking Pylint conf (you can probably consider some stuff as blackbox).

Reply:
https://github.com/saltstack/salt-pylint (Personnaly never tested)

Reply:
I think @jbertran tried it (but I may remember wrong)